pkg/account/oauth-proxy.go

```diff
@@ -20,6 +20,14 @@ func OAuthProxy(jaeger *v1alpha1.Jaeger) *v1.ServiceAccount {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      OAuthProxyAccountNameFor(jaeger),
 			Namespace: jaeger.Namespace,
+			Labels: map[string]string{
+				"app":                          "jaeger",
+				"app.kubernetes.io/name":       OAuthProxyAccountNameFor(jaeger),
```

Comment:
It would be great to have a util function which creates these labels, it seems like a repetitive patern